src/couch/test/couch_stream_tests.erl

```diff
@@ -14,10 +14,11 @@
 
 -include_lib("couch/include/couch_eunit.hrl").
 
+-define(ENGINE, {couch_bt_engine_stream, {Fd, []}}).
```

Comment:
The variable name is hard-coded in a define. Could we do something about it? For example:

`-define(ENGINE(Var), {couch_bt_engine_stream, {Var, []}}).`